react-native.config.js

```diff
@@ -6,6 +6,16 @@ module.exports = {
 			},
 			android: {
 				sourceDir: './platforms/android'
+			},
+			windows: {
```

Comment:
Just curious, did you try creating an app and verifying that the autolinking settings specified here worked? I "think" you can get away with just:

```json
sourceDir: './platforms/windows',
```

I don't think autolinking will have any trouble identifying your solution and project once you've set the sourceDir.

Reply:
FYI, there isn't anything wrong with specifying more, but I think you can get away with:

```
windows: {
  sourceDir: './platforms/windows',
}
```

Reply:
Yep, I have tested auto linking with the current config. Should I remove the rest and test it?